snap/hooks/connect-plug-host-hunspell

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+mkdir -p $SNAP_COMMON/host-hunspell
+snapctl mount --persistent -o ro,bind,noatime,noexec /usr/share/hunspell $SNAP_COMMON/host-hunspell
```

Comment:
Where is that file sourced/executed? Shouldn't it be part of the post-refresh hook instead?

Reply:
No, I was told it needs to be done this way

Reply:
It's executed when `host-hunspell` plug gets connected

Reply:
Ack. The doc was hidden [here](https://snapcraft.io/docs/interface-hooks). What's not clear from the doc is whether that hook is executed everytime the snap is refresh (because if not, the `post-refresh` hook will have disconnected the plug after the first run).

Reply:
No it's only once when connect is done, as much as I understand